arches/app/models/migrations/11613_nodegroup_grouping_node.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 5.1.3 on 2024-11-19 09:29
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("models", "11408_loadstaging_sortorder"),
+    ]
+
+    def add_grouping_node(apps, schema_editor):
+        NodeGroup = apps.get_model("models", "NodeGroup")
+        nodegroups_with_nodes = NodeGroup.objects.filter(node__gt=0).distinct()
+        for nodegroup in nodegroups_with_nodes:
+            nodegroup.grouping_node_id = nodegroup.pk
+        NodeGroup.objects.bulk_update(nodegroups_with_nodes, ["grouping_node_id"])
+
+        PublishedGraph = apps.get_model("models", "PublishedGraph")
+        published_graphs = PublishedGraph.objects.all()
+        for published_graph in published_graphs:
+            for node_dict in published_graph.serialized_graph["nodes"]:
+                node_dict["grouping_node_id"] = node_dict["nodegroup_id"]
```

Comment:
I think the error that I am seeing is caused here. The "grouping_node_id" is getting assigned back to the node. Shouldn't it be assigned to the nodegroup?

Reply:
Thanks, should have retested this. Fixed.

Reply:
Ouch, taking a look.

Reply:
[383bfb2](https://github.com/archesproject/arches/pull/11614/commits/383bfb24f2c1c05850c32b8080c76eb0459c2d15)